modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -982,23 +982,67 @@ def validate_federal_activation_values(service_information)
 
         form_obj_desc = '/serviceInformation/federalActivation'
 
+        # For a valid BDD EP code to be assigned we need these values
+        validate_required_values_for_federal_activation(federal_activation_date, anticipated_separation_date)
+
+        validate_federal_activation_activation_date(federal_activation_date, form_obj_desc)
```

Comment:
well the object is federalActivation and the property is activateDate.  So that was intentional, but it is kind of odd, I could  try to find something else.